index.js

```diff
@@ -123,37 +114,42 @@ class Main {
   }
 
   async render() {
-    screen.key(['escape', 'q', 'C-c'], () => process.exit(0));
-    // fixes https://github.com/yaronn/blessed-contrib/issues/10
-    screen.on('resize', () => {
-      this.bar.emit('attach');
-      this.table.emit('attach');
-      // errorsLine.emit('attach');
-      this.titleBox.emit('attach');
-      this.invocationsLineGraph.emit('attach');
-      this.map.emit('attach');
-      this.log.emit('attach');
-    });
-    screen.title = 'sls-dev-tools';
-
     await this.table.rows.on('select', (item) => {
-      this.funcName = item.content.split('   ')[0];
-      this.refetch();
```

Comment:
do we not want to call the api as soon as we select, waiting for next automatic call seems like the UI would seem laggy.

Reply:
yes done